### PR TITLE
Despecialize Jobs And Events

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

A common complaint I have gotten from server hosts looking to setup servers on our codebase is that we have all of these leftover job and event modifications from DeltaV, which serve a purpose not present in our repository. I've taken the liberty of cleaning up and uncommenting a majority of events that DeltaV commented out for one reason or another at various times, and have also removed every single instance of pre-existing job playtime and whitelist requirements, as well as the antag factors. These especially are a point of contention for anyone wishing to setup a new server, because it's quite literally impossible for anyone on a brand new server that has only been existing for 20 minutes so far, to have the ~200 hours of playtime + whitelist that DeltaV would have required them to have to play Captain. 

This update in turn leaves the onus on server hosts to set their own desired whitelist requirements as needed, rather than enforce DeltaV's unusually strict requirements that we inherited. Furthermore, the sheer number of commented out events from DeltaV was giving server hosts the false impression that we were lacking content compared to Wizden, when truthfully, our predecessor disabled a majority of the game's content, and it was simply never re-enabled. The only events I left commented out are the Terminator(which in our codebase has been superceded by the much better Paradox Anomaly), and the GlimmerWispSpawn, which remains commented due to Glimmer Wisps not yet being rebased.

### CODE REVIEW

File: Resources/Prototypes/GameRules/events.yml [hunk 1]
```diff
@@ -67,24 +67,24 @@
     duration: 1
   - type: ClericalErrorRule
 
-#- type: entity
-#  parent: BaseGameRule # DeltaV - this shit is busted most of the time
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:71]: Very bad; this shit is indeed busted, remember when it caused a CTD when getting gibbed?

File: Resources/Prototypes/GameRules/events.yml [hunk 2]
```diff
@@ -165,14 +165,14 @@
   - type: StationEvent
     earliestStart: 30
     weight: 7.5
-    minimumPlayers: 10 #Enough to hopefully have at least one engineering guy
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:168]: Not sure this is necessary

File: Resources/Prototypes/GameRules/events.yml [hunk 3]
```diff
@@ -165,14 +165,14 @@
...
+    minimumPlayers: 10
     startAnnouncement: station-event-meteor-swarm-start-announcement
     endAnnouncement: station-event-meteor-swarm-end-announcement
     startAudio:
       path: /Audio/Announcements/meteors.ogg
       params:
         volume: -4
-    duration: null #ending is handled by MeteorSwarmRule
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:175]: There's a good reason for this comment

File: Resources/Prototypes/GameRules/events.yml [hunk 4]
```diff
@@ -241,17 +241,17 @@
     maxDuration: 120
   - type: PowerGridCheckRule
 
-# - type: entity # DeltaV - Removes the random sentience.
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:244]: This weight needs to be dropped to 1 or something; sentient vending machines are 0 RP.

File: Resources/Prototypes/GameRules/events.yml [hunk 5]
```diff
@@ -441,20 +441,20 @@
     sounds:
       collection: Paracusia
 
-#- type: entity # DeltaV - Why does this exist??
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:444]: This is mid. Probably should stay removed. But :shrug:

File: Resources/Prototypes/Nyanotrasen/Roles/Jobs/Epistemics/forensicmantis.yml [hunk 6]
```diff
@@ -3,22 +3,14 @@
...
-  requirements:
-    - !type:OverallPlaytimeRequirement
-      time: 18000
-    - !type:DepartmentTimeRequirement
-      department: Epistemics # DeltaV - Epistemics Department replacing Science
-      time: 3600
   startingGear: ForensicMantisGear
   icon: "JobIconForensicMantis"
   supervisors: job-supervisors-rd
-  antagAdvantage: 5 # DeltaV - From 4 to 5
```
Comment (DangerRevolution) [inline Resources/Prototypes/Nyanotrasen/Roles/Jobs/Epistemics/forensicmantis.yml:15]: Why?

File: Resources/Prototypes/Roles/Jobs/Cargo/salvage_specialist.yml [hunk 7]
```diff
@@ -3,13 +3,6 @@
   name: job-name-salvagespec
   description: job-description-salvagespec
   playTimeTracker: JobSalvageSpecialist
-  antagAdvantage: 3 # DeltaV - Reduced TC: External Access + Free hardsuit and weapons
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Cargo/salvage_specialist.yml:6]: Why?

File: Resources/Prototypes/Roles/Jobs/Civilian/service_worker.yml [hunk 8]
```diff
@@ -3,21 +3,17 @@
...
   description: job-description-serviceworker
   playTimeTracker: JobServiceWorker
-  requirements:
-    - !type:OverallPlaytimeRequirement
-      time: 7200 # DeltaV - 2 hours
   startingGear: ServiceWorkerGear
   icon: "JobIconServiceWorker"
   supervisors: job-supervisors-service
-  canBeAntag: true # DeltaV - Can be antagonist
-  antagAdvantage: 1 # DeltaV - Reduced TC: Accesses
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Civilian/service_worker.yml:13]: Why?

File: Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml [hunk 9]
```diff
@@ -34,25 +18,20 @@
   - Janitor
   - Theatre
   - Kitchen
-#  - Chapel
+  - Chapel
   - Hydroponics
   - External
   - Cryogenics
-  # I mean they'll give themselves the rest of the access levels *anyways*.
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Command/head_of_personnel.yml:41]: Bringing back HoP AA needs to be in a different PR.

File: Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml [hunk 10]
```diff
@@ -3,38 +3,28 @@
   name: job-name-paramedic
   description: job-description-paramedic
   playTimeTracker: JobParamedic
-  antagAdvantage: 2 # DeltaV - Reduced TC: External Access
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Medical/paramedic.yml:6]: Why?

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Comment by DangerRevolution** [comment]
HoP AA and TC Disadvantage removal needs to have it's own PR's if intended.

File: Resources/Prototypes/GameRules/events.yml [hunk 11]
```diff
@@ -241,17 +241,17 @@
...
-#     startAudio:
-#       path: /Audio/Announcements/attention.ogg
-#   - type: RandomSentienceRule
+- type: entity
+  id: RandomSentience
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    weight: 10
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:250]: Just going to say this again; this is an insanely 0RP event, vending machines talking isn't as fun as one imagined an having this take up and event slot is BAD.

This needs to have lower weight.
```suggestion
    weight: 1
```

File: Resources/Prototypes/GameRules/events.yml [hunk 12]
```diff
@@ -441,20 +441,20 @@
...
+- type: entity
+  id: ImmovableRodSpawn
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    startAnnouncement: station-event-immovable-rod-start-announcement
+    startAudio:
+      path: /Audio/Announcements/attention.ogg
+    weight: 5
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:453]:
```suggestion
    weight: 0.5
```
Same as before, an immovable rod is 0 RP and shouldn't be taking up an event slot unless there's a good roll for it

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Comment by VMSolidus** [comment]
Yea, that's all good. I pretty wholeheartedly agree with you that RandomSentience & ImmovableRod should be *extremely rare*. Having that ultra low 0.5 weight for both is fine.

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Comment by DangerRevolution** [comment]
Will review tonight unless someone beats me to it

File: Resources/Prototypes/GameRules/events.yml [hunk 13]
```diff
@@ -4,7 +4,7 @@
   noSpawn: true
   components:
   - type: StationEvent
-    weight: 12 # DeltaV - was 10
+    weight: 10
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:7]: Why does this change the comment and the value?

File: Resources/Prototypes/GameRules/events.yml [hunk 14]
```diff
@@ -376,40 +376,40 @@
...
   components:
   - type: StationEvent
-    earliestStart: 90 # DeltaV - was 50
+    earliestStart: 90
     minimumPlayers: 15
-    weight: 2.5 # DeltaV - was 5
+    weight: 2.5
     duration: 1
   - type: ZombieRule
-    minStartDelay: 0 #let them know immediately
```
Comment (DangerRevolution) [inline Resources/Prototypes/GameRules/events.yml:409]: this comment has a reason for being there

File: Resources/Prototypes/Nyanotrasen/Roles/Jobs/Wildcards/martialartist.yml [hunk 15]
```diff
@@ -3,24 +3,20 @@
...
-  - Theatre # DeltaV - Add Theatre access
-  - Boxer # DeltaV - Add Boxer access
+  - Theatre
+  - Boxer
   special:
   - !type:AddComponentSpecial
     components:
     - type: Boxer
       modifiers:
-        coefficients: # These only apply to unarmed
```
Comment (DangerRevolution) [inline Resources/Prototypes/Nyanotrasen/Roles/Jobs/Wildcards/martialartist.yml:23]: This comment has a reason to be here...

File: Resources/Prototypes/Roles/Jobs/Civilian/musician.yml [hunk 16]
```diff
@@ -3,16 +3,13 @@
...
   description: job-description-musician
   playTimeTracker: JobMusician
-  requirements:
-    - !type:OverallPlaytimeRequirement
-      time: 7200 # DeltaV - 2 hours
   startingGear: MusicianGear
   icon: "JobIconMusician"
   supervisors: job-supervisors-hire
   access:
-  - Maintenance # TODO Remove maint access for all gimmick jobs once access work is completed
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Civilian/musician.yml:13]: Why are you removing a TODO?

File: Resources/Prototypes/Roles/Jobs/Science/research_director.yml [hunk 17]
```diff
@@ -21,15 +15,15 @@
...
-  - Mantis # DeltaV - Psionic Mantis, see Resources/Prototypes/DeltaV/Access/epistemics.yml
-  - Chapel # DeltaV - Chaplain is in Epistemics
+  - Mantis
+  - Chapel
   - Cryogenics
-  special: # Nyanotrasen - Mystagogue can use the Bible
+  special:
   - !type:AddComponentSpecial
     components:
-    - type: BibleUser # Nyano - Lets them heal with bibles
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Science/research_director.yml:30]: This comment should stay; just remove the Nyano

File: Resources/Prototypes/Roles/Jobs/Wildcards/boxer.yml [hunk 18]
```diff
@@ -3,22 +3,19 @@
...
-  - Boxer # DeltaV - Add Boxer access
-  special: # Nyanotrasen - BoxerComponent, see Content.Server/Nyanotrasen/Abilities/Boxer/Boxer/BoxerComponent.cs
+  - Theatre
+  - Boxer
+  special:
   - !type:AddComponentSpecial
     components:
     - type: Boxer
       modifiers:
-        coefficients: # These only apply to unarmed
```
Comment (DangerRevolution) [inline Resources/Prototypes/Roles/Jobs/Wildcards/boxer.yml:21]: This comment should be here...

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

**Pull request closed without merging**